CodeConverter/CSharp/DeclarationNodeVisitor.cs

```diff
@@ -1407,7 +1407,14 @@ private static ParameterSyntax ConvertOptionalParameter(ParameterSyntax paramete
             var attrListsToRemove = parameter.AttributeLists.SingleOrDefault(aList => aList.Attributes
                .All(a =>
                 {
-                    var attrIdentifier = ((IdentifierNameSyntax)a.Name).Identifier.Text;
+                    string attrIdentifier = string.Empty;
+
+                    if (a.Name is IdentifierNameSyntax identifierNameSyntax) {
```

Comment:
this is the bug fix to invalid cast exception

Reply:
Great, please could you add a test case that fails before this change, and passes after.
Or if you're not sure how, let me know an example code snippet (probably includes an attribute) that caused this and I can help with that.

Reply:
ok, I can try